src/components/layout/dashboard/DashboardContentContainer.tsx

```diff
@@ -116,8 +116,9 @@ const DashboardContentContainer: React.FC<Props> = ({
           )}
           <Box
             sx={{
-              py: { xs: 2 },
-              px: { xs: 0, sm: 1, lg: 4 },
+              pt: 2,
+              pb: 8,
```

Comment:
8 here is because of the 2 that existed before from `py=2` plus `pb=6` that I removed from the children